### PR TITLE
adding main layout designed

### PR DESCRIPTION
**Pull request by benjaglobant**
adding main layout for the UI to the calculator
image

### CODE REVIEW

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -1,18 +1,200 @@
...
-</androidx.constraintlayout.widget.ConstraintLayout>
\ No newline at end of file
+    android:background="@color/background_layout">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        tools:context=".MainActivity">
+
+        <HorizontalScrollView
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml:14]: put a paddingStart here or a marginStart in the TextView so the number will be a little more separated from the begining

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -1,18 +1,200 @@
...
+                android:textColor="@color/textcolor"
+                android:textSize="@dimen/btn_textsize"
+                app:layout_constraintBottom_toBottomOf="parent"
+                app:layout_constraintEnd_toEndOf="parent"
+                app:layout_constraintStart_toStartOf="parent"
+                app:layout_constraintTop_toTopOf="parent" />
+        </HorizontalScrollView>
+
+        <Button
+            android:id="@+id/btn_2"
```
Comment (mauroLonghi) [inline app/src/main/res/layout/activity_main.xml]: Pls rename all the number buttons id, review the android code style please.

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -1,18 +1,200 @@
...
+        <Button
+            android:id="@+id/btn_divide"
+            style="@style/btn_style"
+            android:layout_marginTop="@dimen/btn_margin_top"
+            android:text="@string/btn_text_divide"
+            app:layout_constraintEnd_toEndOf="@+id/horizontal_scroll_text_view"
+            app:layout_constraintStart_toEndOf="@+id/btn_multiply"
+            app:layout_constraintTop_toBottomOf="@+id/horizontal_scroll_text_view" />
+
+        <Button
```
Comment (SartiMau) [inline app/src/main/res/layout/activity_main.xml:168]: you can create a new style that inherits form btn_style that have the properties 

android:layout_marginTop="@dimen/btn_margin_top"
android:textAllCaps="false"

for the buttons like plus, multiply, delete, divide, sub and clear

File: app/src/main/res/values/colors.xml [hunk 4]
```diff
@@ -3,4 +3,9 @@
     <color name="colorPrimary">#6200EE</color>
     <color name="colorPrimaryDark">#3700B3</color>
     <color name="colorAccent">#03DAC5</color>
+    <color name="background_btn">#66BB6A</color>
+    <color name="background_layout">#000000</color>
```
Comment (SartiMau) [inline app/src/main/res/values/colors.xml]: #000000 is repeated in three different colors.. make the color_black with that value and the background_layout, textcolor and bordercolor use @color/color_black

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (nicolbidone) [reply app/src/main/res/layout/activity_main.xml:14]: another option is to set a margin to the parent, to give all items some distance to the border

File: app/src/main/res/values/colors.xml (hunk 4 above)
Comment (nicolbidone) [reply app/src/main/res/values/colors.xml]: pls if the color already exists do not create it again, if you need to be more descriptive you can create a style

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (benjaglobant) [reply app/src/main/res/layout/activity_main.xml:14]: I like both options, but i'll use the Mau's one. Thanks Nico anyway

File: app/src/main/res/values/colors.xml (hunk 4 above)
Comment (lucianoeli) [reply app/src/main/res/values/colors.xml]: mmm. I agree with Sarti. its ok to have 3 same colors with different name. 
```
<color name="color_black">#000000</color>
<color name="background_layout">@color/color_black</color>
<color name="textcolor">@color/color_black</color>
<color name="bordercolor">@color/color_black</color>
```

Comment (benjaglobant) [reply app/src/main/res/values/colors.xml]: Done!!

File: app/src/main/res/layout/activity_main.xml (hunk 2 above)
Comment (benjaglobant) [reply app/src/main/res/layout/activity_main.xml]: Done!!

File: app/src/main/res/layout/activity_main.xml (hunk 3 above)
Comment (benjaglobant) [reply app/src/main/res/layout/activity_main.xml:168]: Done!

**Review by lucianoeli — APPROVED** [review]

**Review by mauroLonghi — APPROVED** [review]

**Review by SartiMau — APPROVED** [review]

**Pull request merged**